components/prompt_based_laion_retrieval/src/main.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         """
         self.client = ClipClient(
             url=url,
-            indice_name="laion5B",  #TODO:revert back to laion5b-L-14 after backend correction
+            indice_name="laion5B-L-14",
```

Comment:
fyi: this will only work for the public service atm. @ChristiaensBert did you do the backed modifications?

Reply:
Yes, I reverted the arguments to the public service as well. Seems most logical to include in our example so users can easily test it on a small scale.